content/projects/flotiq.md

```diff
@@ -1,34 +1,102 @@
 ---
 title: Flotiq
+repo: flotiq/flotiq-blog
 homepage: https://flotiq.com/
+repo: flotiq/flotiq-blog
```

Comment:
Looks like this line is duplicated. That halts the site build.
If you de-dupe, I should be able to merge this

Reply:
Fixed, sorry about that.